src/get.js

```diff
@@ -1,18 +1,18 @@
-'use strict';
-import curry from 'lodash.curry';
+import curry from 'lodash.curry'
 const getters = {
-  'milliseconds': date => date.getMilliseconds(),
-  'seconds': date => date.getSeconds(),
-  'minutes': date => date.getMinutes(),
-  'hours': date => date.getHours(),
-  'date': date => date.getDate(),
-  'month': date => date.getMonth() + 1,
-  'year': date => date.getFullYear()
-};
+  'milliseconds': (date) => date.getMilliseconds(),
```

Comment:
Personal preference alert: I like not having to use parentheses when only supplying one argument to an arrow function.